index.html

```diff
@@ -155,43 +156,70 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
-                            <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
-                            <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
-                            <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
-                        </ul>
-                        <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
-                        <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
-                    </ul>
-                </div>
-        </section>
+        
+      
+</section>
+<section class="AboutcardBack">
+    <div >
+        <h2  class="tonic4" >About Myself </h2>
+        <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+    </div>
+    <div class = "social-section">
+        <div class="connectmywork">
+            <h4 > LET’S CONNECT</h4>
+        </div>
+        <div class="social">
+            <ul class="social-icons-mywork">
+                <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="getme">
+        <button class="button1"><a>Get my resume </a>  </button>
+    </div>
+    <div class="aboutprogram">
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+            </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
+    </div>
 </section>
+<section class="cardcontact">
+    <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
+        
+        <div class="sectioncontact">
+            <h2  class="contactme" >Contact me </h2>
+        <p class="contact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
+            needs coding. I’d love to help with it  </p>
+        </div>
```

Comment:
**OPTIONAL** - For semantic purposes, I suggest that you remove this section and place it just outside the `form` tag,  but still inside the `section` tag with `class=cardcontact`, please fix this 👍 👍